src/core/DNft.sol

```diff
@@ -76,6 +76,7 @@ contract DNft is ERC721Enumerable, Owned, IDNft {
       dyad                  = Dyad(_dyad);
       oracle                = IAggregatorV3(_oracle);
       MIN_MINT_DYAD_DEPOSIT = _minMintDyadDeposit;
+      MIN_DYAD_DEPOSIT      = _minMintDyadDeposit.mulWadDown(0.1e18);
```

Comment:
Why reduce this by 10x? Doesn't really matter, just want to make sure it's intentional.